public/marked_entity_files.php

```diff
@@ -0,0 +1,104 @@
+<?php include "templates/header.php"; ?>
+
+<?php
+require_once "../modules/models/marked_entity_file.php";
+require_once "../modules/models/attachment.php";
+require_once "../common.php";
+
+ensure_logged_in();
+
+$marked_entity_id = $_GET["marked_entity_id"] ?? $_POST["marked_entity_id"] ?? null;
+
+if (isset($_POST['submit'])) {
+    $marked_entity_file = new MarkedEntityFile();
+    $marked_entity_file->title = $_POST['title'];
+    $marked_entity_file->description = $_POST['description'];
+    $marked_entity_file->entity_id = $_POST['marked_entity_id'];
+    $marked_entity_file->user_id = $_POST['user_id'];
+
+    // Uploads folder needs to be created in the public/ directory
+    // TODO: Make this more convenient
+    $target_dir = "uploads/";
```

Comment:
should it be in a class/section subdir in uploads?

Reply:
assuming that file_id is some unique ID corresponding to each file, I thought that it'd be okay to do this